src/main/java/file/writer/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package file.writer;
```

Comment:
The FileWriterImpl class should be placed in the `service.impl` package according to the checklist item: Don't keep all logic in a single package.